src/Core/src/Platform/Windows/ContentPanel.cs

```diff
@@ -57,6 +57,12 @@ internal FrameworkElement? Content
 
 			_borderPath?.Arrange(new global::Windows.Foundation.Rect(0, 0, finalSize.Width, finalSize.Height));
 
+			if (_borderStroke is not null && Content is not null)
```

Comment:
With the changes, there are several Border tests failing on Windows.
<img width="322" alt="image" src="https://github.com/user-attachments/assets/c02933c3-9666-482f-a364-f1bdaad7c3ac" />

For example BindingOnStrokeShapeShouldWork:
![image](https://github.com/user-attachments/assets/6fe763ba-d7aa-41ee-b307-a94847e53116)

Could you verify if produce now the expected behavior, and in this case, update test reference snapshots?